ci-operator/step-registry/upi/conf/vsphere/ova/windows/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+- staebler
+- jcpowermac
+- patrickdillon
+- jstuever
+- rvanderp3
```

Comment:
@rvanderp3 and @jcpowermac will suffice here. The others I think are from the installer team and might not be aware of the work we are doing.

Reply:
Adjusted.